src/App.js

```diff
@@ -1,12 +1,25 @@
-import './App.css';
+import React from 'react';
+import { Provider } from 'react-redux';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import store from './redux/configureStore';
+import Animals from './components/Animals/Animals';
+import AnimalDetails from './components/Animals/AnimalDetails';
+import Header from './components/Header/Header';
+// import Dropdown from './components/Animals/Dropdown';
```

Comment:
- [ ] Same here, Please do not leave unused code as a comment in your codebase. this will make your code much more noisy, It is always a good practice to keep your code clean from such comments. 😉 👍🏻